packages/realm/src/Collection.ts

```diff
@@ -101,6 +97,13 @@ export abstract class Collection<
    */
   abstract [Symbol.iterator](): Iterator<T>;
 
+  /**
+   * Checks if this collection has not been deleted and is part of a valid Realm.
+   * @returns `true` if the collection can be safely accessed.
+   * @since 0.14.0
```

Comment:
Are the `@since`s really valuable?  Perhaps we could just forget to migrate these :D

Reply:
Yeah, just a thought.  I wouldn't cry if they weren't there